src/Components/Aside/Aside.scss

```
@@ -0,0 +1,62 @@
+aside {
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: white;
+  width: 500px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 50px;
+  transition: left 0.3s;
+  border: 1px solid rgb(224, 221, 221);
+
+  img {
+    margin-bottom: 50px;
+  }
+  .fa-times {
+    padding: 11px 14px;
+    border-radius: 60px;
+    position: absolute;
+    left: 80px;
+    font-weight: thin;
+  }
+  .fa-times:hover {
+    background-color: rgba(240, 239, 239, 0.829);
+  }
+
+  h1 {
+    font-size: 42px;
+    font-weight: bold;
+    text-decoration: none;
+    margin-bottom: 40px;
+  }
+  .sideBar {
+    padding-top: 30px;
+    height: 100vh;
```

```suggestion
    margin-bottom: 40px;
  }
  
  .sideBar {
    padding-top: 30px;
    height: 100vh;
```
새로운 셀랙터 앞에서는 한줄 띄워주세요

---

네, 전체 scss 확인 했습니다.